packages/extension-api/src/extension-api.d.ts

```diff
@@ -437,6 +437,15 @@ declare module '@podman-desktop/api' {
     // Provider to use for the manifest creation, if not, we will try to select the first one available (similar to podCreate)
     provider?: ContainerProviderConnection;
   }
+
+  // Matches required API parameters of Podman options
+  export interface ManifestPushOptions {
+    destination: string;
+    name: string;
```

Comment:
Could you please add some comments (with /** to be exported to doc) on these properties, it is not obvious to me what they represent.

Reply:
you're right! they represent the podman api same names. i'll add more information.